### ISSUE TITLE
Cannot save bot: `Please correct the errors and try again.'

### ISSUE THREAD

**Issue by fabiank0** [microsoft/BotFramework-Services#328]
## Version
No SDK, dev.botframework.com

## Describe the bug

Bug: Settings in the dev.botframework.com page cannot be changed, leading to an error.

C.f. the discussion in the closed issue https://github.com/microsoft/botframework-sdk/issues/3549
I've decided to open a new issue since the original issue was about a different problem.

As has been pointed out in the discussion above, the problem lies in the property 'MsaAppType':'0' that is being sent with the request, leading to the error {"Errors":[{"Field":"MsaAppType","Message":"'MsaAppType' property cannot be changed."}]}

## To Reproduce
1. Go to dev.botframework.com
2. change something
3. press save changes
4. See error: Please correct the errors and try again

## Expected behavior
Saving succeeds successfully.

## Tracking Status

**Dotnet SDK** [TODO]()
- [ ] PR
- [ ] Merged

**Javascript SDK** [TODO]()
- [ ] PR
- [ ] Merged

**Python SDK** [TODO]()
- [ ] PR
- [ ] Merged

**Java SDK** [TODO]()
- [ ] PR
- [ ] Merged

**Samples** [TODO]()
- [ ] PR
- [ ] Merged

**Docs** [TODO]()
- [ ] PR
- [ ] Merged

**Tools** [TODO]()
- [ ] PR
- [ ] Merged

**Comment by anishprasad01** [comment]
@MatDumoulin 

Thank you for the report. We are investigating.

**Comment by MatDumoulin** [comment]
Hi there!

To follow up on the issue, I think we found why the "Please correct the errors and try again." error was occuring. Through [Microsoft documentation](https://docs.microsoft.com/en-us/previous-versions/azure/bot-service/bot-service-migrate-bot?view=azure-bot-service-3.0), we found that [https://dev.botframework.com/](https://dev.botframework.com/) is not the official way of managing your bots anymore.

The fix for us was to migrate our bots to Azure. Once the migration is done, you can change the messaging endpoint directly in Azure instead of doing it through dev.botframework.com.

I hope others might find this information useful!

As a side note, [the documentation](https://docs.microsoft.com/en-us/previous-versions/azure/bot-service/bot-service-migrate-bot?view=azure-bot-service-3.0) has to be updated since our bots are only using the Microsoft Teams channel but they still needed to be migrated.

**Comment by anishprasad01** [comment]
@MatDumoulin 

We are working on getting that error resolved, but in the long-term migrating to Azure is the best bet. 

> As a side note, the documentation has to be updated since our bots are only using the Microsoft Teams channel but they still needed to be migrated.

I don't understand what you mean by this. Migrating a bot to Azure happens irrespective of channel.

**Comment by bobokids** [comment]
We had a PR ready for this issue. Incident 284309614 : [PR Ready]Need a fix: Cannot save bots in Dev Portal

**Comment by fabiank0** [comment]
Thanks @MatDumoulin. @anishprasad01 Do you still need an App ID of a Bot?

**Comment by anishprasad01** [comment]
Nope, we're good. Once the PR makes its way through and goes live, the problem should be fixed and we'll close this issue.

**Comment by cosmicshuai** [comment]
I have a PR fix this issue.
